README.md

```diff
@@ -1,2 +1,16 @@
 # hw01
-Repository for homework 01
+For the purpose of this assignment, my brief biography will follow a list format. 
+# Uninteresting Bio
+
+* The usual start, name. I am **Dalia**, a visiting PoliSci undergrad from Cairo University. It's alright if you mispronounce my name as *Daria*; that's the name of a 90's cartoon character whose personality bizarrely resembles mine.   
+# What brought me here?
```

Comment:
It's good to start thinking about code styling. This renders correctly, but reading the original `.md` file, everything is a bit blending together. Another way of writing this is the use of blank lines:

```
* The usual start, name. I am **Dalia**, a visiting PoliSci undergrad from Cairo University. It's alright if you mispronounce my name as *Daria*; that's the name of a 90's cartoon character whose personality bizarrely resembles mine.       

# What brought me here?

* To UChicago? A generous Egyptian billionaire was kind enough to send me and 8 other Egyptians on a one-year scholarship program suitably called under his name, The *Sawiris ISAL Program*. 
* To this course in particular? My minor is social science computing yet I have only been exposed to Microsoft Access, Gephi, and Python. I was thus extremely thrilled to take this course to finally be able to conduct statistical analysis for future research. 
```

Which still renders correctly:

* The usual start, name. I am **Dalia**, a visiting PoliSci undergrad from Cairo University. It's alright if you mispronounce my name as *Daria*; that's the name of a 90's cartoon character whose personality bizarrely resembles mine.       

# What brought me here?

* To UChicago? A generous Egyptian billionaire was kind enough to send me and 8 other Egyptians on a one-year scholarship program suitably called under his name, The *Sawiris ISAL Program*. 
* To this course in particular? My minor is social science computing yet I have only been exposed to Microsoft Access, Gephi, and Python. I was thus extremely thrilled to take this course to finally be able to conduct statistical analysis for future research.